### PR TITLE
Feature/skype fix richtext

### PR DESCRIPTION
**Pull request by phillipadsmith**

### CODE REVIEW

File: core/core_integrations/messenger.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+                    genericMessageRequest({
+                        attachment: {
+                            type: 'image',
+                            payload: {
+                                url: image
+                            }
+                        }
+                    }, thread);
+                    break;
+                case 'file':
```
Comment (mrkno) [inline core/core_integrations/messenger.js]: Codacy Issue found: [Expected indentation of 12 space characters but found 16.](https://www.codacy.com/app/theprojectshop/Kassy/file/2626852437/issues/source?bid=3282987&fileBranchId=3316939#l131)

File: core/core_integrations/messenger.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                case 'url':
+                    genericMessageRequest({
+                        attachment: {
+                            type: 'image',
+                            payload: {
+                                url: image
+                            }
+                        }
+                    }, thread);
+                    break;
```
Comment (mrkno) [inline core/core_integrations/messenger.js]: Codacy Issue found: [Expected indentation of 4 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2626852437/issues/source?bid=3282987&fileBranchId=3316939#l130)

File: core/core_integrations/messenger.js [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+        getUsers: function() {
+            return exports.config.users || {};
+        },
+        sendImage: function(type, image, description, thread) {
+            if (description) {
+                this.sendMessage(description, thread);
+            }
+            switch (type) {
+                case 'url':
+                    genericMessageRequest({
```
Comment (mrkno) [inline core/core_integrations/messenger.js]: Codacy Issue found: [Expected indentation of 16 space characters but found 20.](https://www.codacy.com/app/theprojectshop/Kassy/file/2626852437/issues/source?bid=3282987&fileBranchId=3316939#l122)

File: core/core_integrations/messenger.js [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+        },
+        getUsers: function() {
+            return exports.config.users || {};
+        },
+        sendImage: function(type, image, description, thread) {
+            if (description) {
+                this.sendMessage(description, thread);
+            }
+            switch (type) {
+                case 'url':
```
Comment (mrkno) [inline core/core_integrations/messenger.js]: Codacy Issue found: [Expected indentation of 12 space characters but found 16.](https://www.codacy.com/app/theprojectshop/Kassy/file/2626852437/issues/source?bid=3282987&fileBranchId=3316939#l121)

File: core/core_integrations/messenger.js [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+                spl[i] = spl[i].substr(str.length);
+            }
+            spl.splice(i, 1);
+            for (var j = 0; j < strs.length; j++) {
+                spl.splice(i + j, 0, strs[j]);
+            }
+            i += strs.length;
+        }
+
+        for (var i = 0; i < spl.length; i++) {
```
Comment (mrkno) [inline core/core_integrations/messenger.js]: Codacy Issue found: ["i" is already defined](https://www.codacy.com/app/theprojectshop/Kassy/file/2626852437/issues/source?bid=3282987&fileBranchId=3316939#l60)

File: core/core_integrations/messenger.js [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+            var curr = spl[i];
+            for (var j = i + 1; j < spl.length; j++) {
```
Comment (mrkno) [inline core/core_integrations/messenger.js]: Codacy Issue found: ["j" is already defined](https://www.codacy.com/app/theprojectshop/Kassy/file/2626852437/issues/source?bid=3282987&fileBranchId=3316939#l62)

File: core/integrations/integrations.js [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+    return list;
+};
+
+exports.setIntegrationConfigs = function(platform) {
+    for (var i = 0; i < selectedIntegrations.length; i++) {
+        selectedIntegrations[i].instance.platform = platform;
+        selectedIntegrations[i].instance.config = platform.config.loadOutputConfig(selectedIntegrations[i].name);
+
+        for (var name in selectedIntegrations[i].instance.config) {
+            if (name.startsWith("ENV_") && name === name.toUpperCase()) {
```
Comment (mrkno) [inline core/integrations/integrations.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/theprojectshop/Kassy/file/2626852457/issues/source?bid=3282987&fileBranchId=3316939#l65)

File: core/integrations/integrations.js [hunk 8]
```diff
@@ -0,0 +1,146 @@
+﻿/**
```
Comment (mrkno) [inline core/integrations/integrations.js]: Codacy Issue found: [Irregular whitespace not allowed](https://www.codacy.com/app/theprojectshop/Kassy/file/2626852457/issues/source?bid=3282987&fileBranchId=3316939#l1)

**Comment by phillipadsmith** [comment]
Yikes, terrible pull request. Let me just create one from the source file. Ug.

File: core/integrations/integrations.js (hunk 7 above)
Comment (mrkno) [inline core/integrations/integrations.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/theprojectshop/Kassy/file/2626852457/issues/source?bid=3282987&fileBranchId=3316939#l65)

File: core/core_modules/admin.js [hunk 9]
```diff
@@ -0,0 +1,176 @@
+﻿var config = require.once('../config.js'),
```
Comment (mrkno) [inline core/core_modules/admin.js]: Codacy Issue found: [Irregular whitespace not allowed](https://www.codacy.com/app/theprojectshop/Kassy/file/2626852442/issues/source?bid=3282987&fileBranchId=3316939#l1)

**Pull request closed without merging**